s3watcher/lambda/cloudformation.json

```diff
@@ -0,0 +1,3 @@
+{
+  "file_left_intentionally": "blank"
```

Comment:
`node-riffraff-artefact` currently fails without this file. 

Probably should PR another update.